packages/manager/modules/pci/src/projects/creating/module.js

```diff
@@ -22,6 +23,7 @@ angular
   .config(routing)
   .run(/* @ngTranslationsInject:json ./translations */)
   .component(component.name, component)
+  .service('PciProjectsService', PciProjectsService)
```

Comment:
:warning: Service with this name already exists.

```sh
$ git grep "'PciProjectsService'" -- "*.js" 
packages/manager/modules/pci/src/projects/projects.module.js:  .service('PciProjectsService', service)
```

Reply:
Yes @antleblanc, but i think is better to add also this dependency into the module.
For me each module must declare his dependencies.

Good ?

Reply:
@mohammed-zahaf As angularJS base itself on names to identify modules and providers, it is definitely not a good practice to have two services having the same name

Reply:
Removed, don't need it.